docs/server/settings.rst

```diff
@@ -289,8 +289,9 @@ Translation environment configuration settings.
      Added the :setting:`WEIGHT <POOTLE_TM_SERVER-WEIGHT>` option. Also added
      another default TM used to import external translations from files.
 
+  Default: ``{}`` (empty dict)
```

Comment:
Please add a blank line before this, not only for making the diff easier to understand, but because readability counts.